web/templates/login/index.html.eex

```diff
@@ -1,25 +1,34 @@
 <div class="ma3 ma4-m ma4-l">
-  <h3 class="tc">Log in</h3>
+  <h3 class="hl-darkBlue tc">Log in</h3>
   <section class="mw7 center">
-    <%= form_for @conn, login_path(@conn, :create), [as: :login, class: "mw6 bg-white br2 br--bottom shadow-1--bottom pa3 mv3 ba b--black-10 center"], fn f -> %>
+    <div class="bb b--black-20 pb4 mt4">
+      <span class="fl mt0">Not an existing user?</span>
+      <a class="link dim fr mt0 db" href="/users/new">Sign up</a>
+    </div>
+
+    <!-- form -->
```

Comment:
Hmm I'm not a fan of these types of comments, but fine if the team thinks they're useful.